repository/repository-server-config/pom.xml

```diff
@@ -80,7 +87,21 @@
 			<artifactId>commons-compress</artifactId>
 			<version>1.18</version>
 		</dependency>
-
+		
+		<!-- AWS Elastic Search configuration (start) -->
+		<dependency>
+		    <groupId>com.amazonaws</groupId>
+		    <artifactId>aws-java-sdk-core</artifactId>
+		    <version>1.11.574</version>
```

Comment:
Possible to use 1.11.392 instead ? as pre approved CQ available

Reply:
Can try.